website/docs/golioth-exploration/01-golioth-intro/_partials/install_nrf_connect.md

```diff
@@ -7,8 +7,8 @@
       you also need to [install the SEGGER J-Link
       tools](https://www.segger.com/downloads/jlink/#J-LinkSoftwareAndDocumentationPack)
 
-      **Mac M1/M2 Users:** Our testing found that you must use the `Universal
-      Installer` version of the SEGGER tools
+      **Mac users with Apple Silicon:** Our testing found that you must use the
+      `Universal Installer` version of the SEGGER tools with nRF Connect for Desktop.
```

Comment:
Used the generic "Apple Silicon" term because there is now M3 (and will probably be M4, M5, etc)